framework/graphics/dx12_dump_resources.h

```diff
@@ -137,10 +148,7 @@ class Dx12DumpResources
     ~Dx12DumpResources();
 
     void StartDump(const TrackDumpResources& resources);
-    void WriteResource(const CopyResourceData&                             resource_data,
-                       const std::vector<std::pair<std::string, int32_t>>& json_path,
-                       const std::string&                                  file_name,
-                       const std::string&                                  type);
+    void WriteResource(const CopyResourceData* resource_data);
```

Comment:
Should we use CopyResourceDataPtr to replace all CopyResourceData*?

Reply:
It isn't required here but I think that is a good idea for consistency. I will make that update.

Reply:
Done